client/src/components/admin_dashboard/SchoolsList.js

```diff
@@ -18,10 +26,23 @@ export default function SchoolsList(props) {
 
     const pageSize = 4;
 
+    const urlBuilder = (path) => {
```

Comment:
Do we need 3 different urlBuilder functions? I kinda feel like these could all be unified and put into a util file